docs/topics/auth/customizing.txt

```diff
@@ -234,10 +234,12 @@ for example, to control anonymous access.
 Authorization for inactive users
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-An inactive user is a one that is authenticated but has its attribute
-``is_active`` set to ``False``. However this does not mean they are not
-authorized to do anything. For example they are allowed to activate their
-account.
+An inactive user is a one that has its attribute ``is_active`` set to
+``False``. By default, they can't authorize until their accounts will be
+activated. To change this, you need to override
+:meth:`~django.contrib.auth.backends.ModelBackend.authenticate` and
+:meth:`~django.contrib.auth.backends.ModelBackend.get_user`.
+Also, if user has no ``is_active`` field - he will be allowed to login.
```

Comment:
Ok. Also `django.contrib.auth.backends.ModelBackend.get_user` is not documented yet thus it gives a warning during `make spelling`. What would you suggest: avoid to use this reference or to document `get_user` method? Thanks.